docs/concepts/toc.yml

```diff
@@ -13,5 +15,5 @@
     href: ../bookmarks-view.md
   - name: Augmented Reality View
     href: ../ar.md
-- name: API Reference
-  href: ../api/index.md
+- name: API reference
```

Comment:
Shouldn't titles be capitalized?

Reply:
Weirdly the Esri style is that headings/titles are not capitalized.